src/translations/de.json

```diff
@@ -33,6 +33,25 @@
   "AuthenticationPage.verifyEmailTitle": "{name}, bitte überprüfe deinen E-Mail-Posteingang, um deine E-Mail-Adresse zu bestätigen",
   "Avatar.bannedUserDisplayName": "Gesperrter Benutzer",
   "Avatar.deletedUserDisplayName": "Gelöschter Benutzer",
+  "BatchEditListingUploaderPanel.title": "Fotos hochladen",
```

Comment:
None of this is needed. But did you add this on the Sharetrive console?

If not let's touch base on how to do it because it is tricky